package.json

```diff
@@ -35,35 +36,32 @@
     "grunt-contrib-jshint": "^0.11.2",
     "grunt-contrib-uglify": "^4.0.1",
     "grunt-contrib-watch": "^1.1.0",
+    "grunt-eslint": "^22.0.0",
     "grunt-exorcise": "^2.1.1",
     "grunt-ftp-push": "^1.2.1",
     "grunt-githash": "^0.1.3",
-    "grunt-githooks": "^0.6.0",
-    "grunt-jscs": "^2.5.0",
     "grunt-jsdoc": "^2.4.0",
-    "grunt-mocha-istanbul": "^5.0.2",
+    "grunt-mocha-cli": "^5.0.0",
+    "grunt-simple-nyc": "^3.0.1",
     "grunt-string-replace": "^1.3.1",
     "ink-docstrap": "^1.3.2",
     "intern": "3.4.6",
-    "iron-mocha": "^1.0.3",
     "iron-node": "^3.0.23",
-    "istanbul": "^0.4.5",
     "jsdoc": "^3.6.2",
     "jsdom": "^13.2.0",
     "load-grunt-tasks": "^4.0.0",
-    "mocha": "^2.3.4",
+    "mocha": "^7.0.0",
+    "nyc": "^15.0.1",
     "sinon": "^7.3.2",
     "time-grunt": "^2.0.0",
     "uglify-js": "^3.6.0"
   },
   "dependencies": {
     "codem-isoboxer": "0.3.6",
     "fast-deep-equal": "2.0.1",
-    "imsc": "^1.0.2",
     "html-entities": "^1.2.1",
-    "localforage": "^1.7.1",
-    "request": "^2.87.0",
-    "request-promise": "^4.2.2"
+    "imsc": "^1.0.2",
```

Comment:
update to version 1.1.1 of imsc could be a good idea... ;-)

Reply:
thanks, will put this on the todo list, too

Reply:
aj, ver 1.1.1 has obviously breaking changes to our unit tests. anyway, i accept the challenge ;)

Reply:
If imsc lib is updated, functional/manual tests for streams with subtitles must be carried out thoroughly